SCRAPERS-LIST.md

```diff
@@ -416,6 +416,7 @@ freeusemilf.com|PaperStreetMedia.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 freshoutofhighschool.com|NewSensationsNetworkSites.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 frolicme.com|FrolicMe.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 fuckedfeet.com|VNAGirls.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
+fuckedhard18.com|FuckedHard18.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
```

Comment:
Your scraper only support scene so you should remove the ❌ for Gallery & Movies
`fuckedhard18.com|FuckedHard18.yml|:heavy_check_mark:|:x:|-|-|-|-`